.github/workflows/ci-codegen.yaml

```diff
@@ -96,3 +96,11 @@ jobs:
           npm run gar-login
           npm publish
         working-directory: ee/codegen
+
+      - name: Bump codegen-service
+        env:
+          VELLUM_AUTOMATION_APP_ID: ${{ secrets.VELLUM_AUTOMATION_APP_ID }}
+          VELLUM_AUTOMATION_PRIVATE_KEY: ${{ secrets.VELLUM_AUTOMATION_PRIVATE_KEY }}
+          VELLUM_AUTOMATION_INSTALLATION_ID: ${{ secrets.VELLUM_AUTOMATION_INSTALLATION_ID }}
```

Comment:
shoutout to @meikemagdalena for the help here 💥

Reply:
🕺 🪩 💃